features/step_definitions/networking.rb

```diff
@@ -745,4 +777,12 @@
     @result = pod.exec(network_cmd, as: admin)
     raise "Failed to execute network command!" unless @result[:success]
   end
-end
\ No newline at end of file
+end
+
+Given /^the env is using "([^"]*)" networkType$/ do |network_type|
+  ensure_admin_tagged
+  _admin = admin
+  step "I select a random node's host"
```

Comment:
is this step needed?

Reply:
I believe this will be used as a gating in upcoming OVN specific test cases automation and will always be a check to make sure cluster we are using is indeed on OVN

Reply:
aha,misunderstood. I mean `step "I select a random node's host"` is necessary for this